tests/sentry/incidents/models/test_alert_rule.py

```diff
@@ -211,6 +211,35 @@ def test_conditionally_subscribing_project_initializes_activation(self):
             assert current_activation.query_subscription == sub
             assert current_activation.is_complete() is False
 
+    def test_get_for_metrics(self):
```

Comment:
We should test for prohibited access patterns here - test whether this is guarded against cross-org/cross-project access.

Reply:
That is already being done in the test, unless I am missing smth?

Reply:
Implemented in the test with endpoints